dragons/meraxes/io.py

```diff
@@ -879,12 +891,17 @@ def read_grid(fname, snapshot, name, h=None, h_scaling={}):
     return grid
 
 
-def list_grids(fname, snapshot):
+def list_grids(spec, fname, snapshot):
```

Comment:
For all of the functions that now expected the `spec` argument. Can you add a check that will throw an exception if `spec not in (0, 1)`?

Reply:
Sure thing!